test/arrays.js

```diff
@@ -1,6 +1,7 @@
 (function() {
+  var _ = typeof require == 'function' ? require('../underscore') : window._;
```

Comment:
It was an issue with that cli exposing all the underscore methods to global scope